imperial_coldfront_plugin/gpfs_client.py

```diff
@@ -205,6 +211,39 @@ def set_quota(
             blockGracePeriod="null",
         )
 
+    @check_job_status
+    @json
+    @post("filesystems/{filesystemName}/filesets/{filesetName}/directory/{path}")
+    def _create_fileset_directory(
+        self,
+        filesystemName: str,
+        filesetName: str,
+        path: str,
+        **data: Body,
+    ) -> requests.Response:
+        pass
+
+    def create_fileset_directory(
+        self, filesystem_name: str, fileset_name: str, path: str
+    ):
+        """Create a new directory within a fileset.
+
+        Args:
+          filesystem_name: name of the filesystem containing the fileset.
+          fileset_name: name of the fileset in which to create the directory.
+          path: location of the directory as a relative path w.r.t the fileset path,
+            will create new directories recursively as required.
+        """
+        return self._create_fileset_directory(
+            filesystem_name,
+            fileset_name,
+            path,
+            user="root",
+            group="root",
```

Comment:
Are these values placeholders or will always be the same?

Reply:
Unsure at this point.